sympy/functions/combinatorial/factorials.py

```diff
@@ -727,11 +729,15 @@ class FallingFactorial(CombinatorialFunction):
     def eval(cls, x, k):
         x = sympify(x)
         k = sympify(k)
+        xint, kint = [int_like(i) for i in (x, k)]
```

Comment:
I think the second arg should be a strict integer and something like `ff(5, 2.0)` should raise. At `live.sympy` `ff(5,2.0)` remains unevaluated but evalf evaluates to 20.0...hmm. 

But we really don't want to add more to this PR. Only what is necessary for `monomial` -- and all the `binomial` changes are needed to allow `multinomial(i,j) == multinomial(j,i)`